src/config.cpp

```diff
@@ -42,23 +43,41 @@ bool GlobalConfig::UseCashAddrEncoding() const {
     return useCashAddr;
 }
 
+void GlobalConfig::SetEnableMinerFund(bool c) {
+    enableMinerFund = c ? MinerFundStatus::Enabled : MinerFundStatus::Disabled;
+}
+bool GlobalConfig::EnableMinerFund() const {
+    return enableMinerFund == MinerFundStatus::Unset
+               ? GetChainParams().GetConsensus().enableMinerFund
+               : enableMinerFund == MinerFundStatus::Enabled;
+}
+void GlobalConfig::SetExcessUTXOCharge(Amount fee) {
+    excessUTXOCharge = fee;
+}
+
+Amount GlobalConfig::GetExcessUTXOCharge() const {
+    return excessUTXOCharge;
+}
+
 DummyConfig::DummyConfig()
-    : chainParams(CreateChainParams(CBaseChainParams::REGTEST)) {}
+    : chainParams(CreateChainParams(CBaseChainParams::REGTEST)),
+      enableMinerFund(chainParams->GetConsensus().enableMinerFund) {}
 
 DummyConfig::DummyConfig(std::string net)
-    : chainParams(CreateChainParams(net)) {}
+    : chainParams(CreateChainParams(net)),
+      enableMinerFund(chainParams->GetConsensus().enableMinerFund) {}
 
 DummyConfig::DummyConfig(std::unique_ptr<CChainParams> chainParamsIn)
-    : chainParams(std::move(chainParamsIn)) {}
+    : chainParams(std::move(chainParamsIn)),
+      enableMinerFund(chainParams->GetConsensus().enableMinerFund) {}
 
 void DummyConfig::SetChainParams(std::string net) {
     chainParams = CreateChainParams(net);
 }
 
-void GlobalConfig::SetExcessUTXOCharge(Amount fee) {
```

Comment:
Why is this being removed?

Reply:
Should not have been, but it worked apparently because Amaury ripped out all the other code using it.